builder-hex0.hex2

```diff
@@ -64,7 +64,7 @@
 
 #------------------------------------------------------------
 # Memory:
-# 54000000 - BFFFFFFF files (~1812MB)
+# 54000000 - BBFFFFFF files (~1744MB)
```

Comment:
This is not actually a change, it just clarifies that 0xBC000000-0xBFFFFFFF might not be a usable memory range on all systems, due to BIOS using it for various purposes, e.g. ACPI.